flink-libraries/flink-gelly/src/test/java/org/apache/flink/graph/test/operations/DegreesWithExceptionITCase.java

```diff
@@ -38,29 +32,10 @@
  * Test expected errors for {@link Graph#inDegrees()},
  * {@link Graph#outDegrees()}, and {@link Graph#getDegrees()}.
  */
-public class DegreesWithExceptionITCase extends TestLogger {
+public class DegreesWithExceptionITCase extends AbstractTestBase {
 
 	private static final int PARALLELISM = 4;
```

Comment:
I think this could be replaced in all the tests by making `DEFAULT_PARALLELISM` in `AbstractTestBase` protected and using that instead.